packages/shared/routes/dashboard/wallet/views/Send.svelte

```diff
@@ -1,8 +1,9 @@
 <script lang="typescript">
     import { getContext, onDestroy, onMount } from 'svelte'
     import { get, Readable } from 'svelte/store'
+    import { ActionSheet, ActionSheetButtonStyle } from '@capacitor/action-sheet'
```

Comment:
This should be implemented on `platform.ts`, `capacitorAPI.ts`, `electronAPI.js` as here with share plugin:
https://github.com/iotaledger/firefly/pull/3499/files